Form/DataTransformer/MoneyToLocalizedStringTransformer.php

```diff
@@ -19,20 +21,20 @@
  */
 class MoneyToLocalizedStringTransformer extends NumberToLocalizedStringTransformer
 {
-	/** @var string ISO currency code **/
-	private $currencyCode;
+	/** @var \Money\Currency **/
+	private $currency;
 	/** @var \Money\Currencies|null */
 	private $currencies;
 
 	/**
-	 * @param string                 $currencyCode ISO currency code
+	 * @param Currency               $currency
 	 * @param int                    $scale
 	 * @param bool                   $grouping
 	 * @param \Money\Currencies|null $currencies
 	 */
-	public function __construct($currencyCode, $scale, $grouping, Currencies $currencies = null)
+	public function __construct($currency, $scale, $grouping, Currencies $currencies = null)
```

Comment:
Please add type hint `Currency` to constructor.

Reply:
Changing public method signature is BC break, so I'll throw deprecation notice if string is passed, you'll add hint when 2.0 is ready

Reply:
OK.